owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/utils/Log_OC.java

```diff
@@ -174,15 +174,12 @@ synchronized private static void appendLog(String text) {
                 isMaxFileSizeReached = false;
             }
 
-	        String timeStamp = new SimpleDateFormat(SIMPLE_DATE_FORMAT).format(Calendar.getInstance().getTime());
+	        String timeStamp = new SimpleDateFormat(SIMPLE_DATE_FORMAT, Locale.ENGLISH).format(Calendar.getInstance().getTime());
 
 	        try {
 	            mBuf = new BufferedWriter(new FileWriter(mLogFile, true));
 	            mBuf.newLine();
-	            mBuf.write(timeStamp);
-	            mBuf.newLine();
-	            mBuf.write(text);
-	            mBuf.newLine();
+	            mBuf.write(timeStamp+" "+text);
```

Comment:
Some spaces here between timeStamp and the rest of parts of this String.

Reply:
Btw, I applied codestyle for complete file, as you see a lot of places are not well formatted. 
I see the room for a general solution, what's about other lines.
Anyway, I applied request